tests/test_quantumGraph.py

```diff
@@ -316,7 +307,14 @@ def testIsConnected(self):
         self.assertFalse(self.qGraph.isConnected)
         # make a broken subset
         filteredNodes = [n for n in self.qGraph if n.taskDef.label != "U"]
-        subset = self.qGraph.subset((filteredNodes[0], filteredNodes[1]))
+        label0 = filteredNodes[0].taskDef.label
+        node1 = None
+        for n in filteredNodes[1:]:
+            if n.taskDef.label != label0:
+                node1 = n
+                break
+        self.assertIsNotNone(node1, f"Could not find node with label other than {label0}")
+        subset = self.qGraph.subset((filteredNodes[0], node1))
```

Comment:
OK, so my comment on this is that this test should just be taken out completely.

Move `self.assertFalse(self.qGraph.isConnected)` to the beginning of the `testSubsetToConnected` test. All that is being tested in `testIsConnected` is tested in `testSubsetToConnected` anyway, so no need to hit the same methods more than once, and `subsetToConnected` is the proper way to split up a graph anyway.